lib/data-types.js

```diff
@@ -465,7 +465,7 @@ DATE.prototype._stringify = function _stringify(date, options) {
 function DATEONLY() {
   if (!(this instanceof DATEONLY)) return new DATEONLY();
 }
-util.inherits(DATEONLY, ABSTRACT);
+inherits(DATEONLY, ABSTRACT);
```

Comment:
Is this needed for the JSON to work? Could you do this in a seperate PR? The commits will get squashed